lib/client.js

```diff
@@ -182,6 +182,54 @@ Client.prototype._defineMethod = function(method, location) {
   };
 };
 
+Client.prototype._isSequenceRequired = function(methodName) {
+  var tns = this.wsdl.definitions.$targetNamespace;
+  var methodRequestName = _.result(this.wsdl.definitions, 'messages.'+methodName+'.$name');
+  var args = _.result(this.wsdl.definitions, 'messages.'+methodRequestName+'.parts');
+  if(typeof args === 'undefined' && typeof _.pick(args, 'params') !== 'undefined') {
+    return false;
+  }
+  if(Object.keys(args).length === 1) {
+    return false;
+  }
+  var complexTypeName = _.result(this.wsdl.definitions, 'messages.'+methodRequestName+'.element.$name');
+  var modeOfComplexType = _.result(this.wsdl.definitions, 'schemas[\''+tns+'\'].elements.'+complexTypeName+'.children[0].children[0].name');
```

Comment:
can you break this up so the line is shorter?  same comment with spaces around operators